src/sources/dataModel/taxonomic-service.json

```diff
@@ -210,52 +429,104 @@
                 },
                 "schema:url": {
                     "type": "string",
-                    "description": "URL of the item that leads to the point of contact",
-                    "examples": [
-                        "cetaf.org"
-                    ]
-                },
-                "schema:sameAs": {
-                    "type": "string",
-                    "description": "URL of a reference Web page that unambiguously indicates the item's identity. E.g. the URL of the item's Wikipedia page, Wikidata entry, or official website.",
+                    "description": "URL to a contact webpage or contact form for the resource.",
                     "examples": [
                         "cetaf.org"
                     ]
                 }
             },
             "additionalProperties": false
         },
-        "schema:Maintainer": {
+        "schema:Author": {
             "type": "array",
-            "description": "An array of maintainers of a Dataset, software package (SoftwareApplication), or other Project. A maintainer is a Person or Organization that manages contributions to, and/or publication of, some (typically complex) artefact.",
+            "description": "An array of authors of this service.",
             "items": {
                 "type": "object",
                 "properties": {
                     "schema:identifier": {
+                        "@id": "orcid",
```

Comment:
This doesn't look right. Looks like you want the orcid here. This is a string with a specific format. Although your example is a github name.